pyproject.toml

```diff
@@ -68,6 +68,7 @@ neuronx = ["optimum[neuronx]"]
 mamba = ["mamba_ssm", "causal-conv1d==1.0.2"]
 math = ["sympy>=1.12", "antlr4-python3-runtime==4.11"]
 multilingual = ["nagisa>=0.2.7", "jieba>=0.42.1", "pycountry"]
+openai = ["openai"]
```

Comment:
We don't really use this package

Reply:
I looked at the code - indeed - this package is not used anywhere. I removed the `lm_eval[openai]` option from the installation